### PR TITLE
Added Packet Sniffer

### PR DESCRIPTION
**Pull request by DJAcquila**
**Changes:**
 Add in the networking directory a Packet Sniffer, made with python.

### CODE REVIEW

File: code/networking/src/PacketSniffer/packetSniffer.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+	checksum = ip_header[5]
+	end_maquina_saida = socket.inet_ntoa(ip_header[6])
+	end_maquina_destino = socket.inet_ntoa(ip_header[7])
+
+	if protocolo == 17:
+		prox = "UDP"
+	elif protocolo == 6:
+		prox = "TCP"
+	else:
+		prox = 'nul'
```
Comment (arnavb) [inline code/networking/src/PacketSniffer/packetSniffer.py]: PEP8 says to use spaces, so use spaces.

File: code/networking/src/PacketSniffer/packetSniffer.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+	print ("\tMAC address de source:\t%s:%s:%s:%s:%s:%s" % (end_maquina_saida[0:2], end_maquina_saida[2:4], end_maquina_saida[4:6], 			end_maquina_saida[6:8], end_maquina_saida[8:10], end_maquina_saida[10:12]))
+	print ("\tProtocolo:\t%s" %hex(protocolo))
+
+	#IPv4 == 0X0800
+	
+	if protocolo == 8:
+		ip = True
+	recv = recv[14:]
+	
+	return recv, ip
```
Comment (arnavb) [inline code/networking/src/PacketSniffer/packetSniffer.py]: Add a newline below.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_cosmos

**Comment by DJAcquila** [comment]
Thanks ;D

**Pull request merged**